core/src/main/scala/io/finch/request/package.scala

```diff
@@ -162,6 +180,8 @@ package object request {
       def apply[Req](req: Req)(implicit ev: Req => HttpRequest) =
         try number.toFuture
         catch { case _: IllegalArgumentException => ValidationFailed(param, rule).toFutureException }
+
+      override def toString: String = s"param: $param"
```

Comment:
Two more reasons against the `suchThat` syntax:

2) it looks like repetition where many users might wonder why they need to repeat the param name

3) how would you write a `suchThat` method on a reader for more than one param then?